src/main/java/frc/robot/RobotMap.java

```diff
@@ -1,4 +1,5 @@
 package frc.robot;
 
 public class RobotMap {
+    public static final int LEFT_ELEVATOR_CAN_ID = Constants.PLACEHOLDER_INT, RIGHT_ELEVATOR_CAN_ID = Constants.PLACEHOLDER_INT;
```

Comment:
rename to LEFT_ELEVATOR_MOTOR_CAN_ID (do the same for the right one too). this adds specificity.